### PR TITLE
add --trace-http hidden flag to set TraceOn to minio client

### PR DESCRIPTION
**Pull request by fatpat**

### CODE REVIEW

**Review by klauspost — APPROVED** [review]
LGTM

**Review by klauspost — CHANGES_REQUESTED** [review]
Sorry for changing my mind. Could we make it `--debug` - that way it matches our other tools.

File: cli/flags.go [hunk 1]
```diff
@@ -217,4 +217,9 @@ var ioFlags = []cli.Flag{
 		Usage:  "Disable HTTP Keep-Alive",
 		Hidden: true,
 	},
+	cli.BoolFlag{
+		Name:   "trace-http",
```
[the remark this answers is not in the document]
Comment (fatpat) [reply cli/flags.go:221]: there is already a `--debug` flag. Either we use it and set `TraceOn` when `--debug` is set or use another one like `--debug-http`.

what do you prefer ?

[the remark this answers is not in the document]
Comment (harshavardhana) [reply cli/flags.go:221]: The same debug flag can be used @fatpat

[the remark this answers is not in the document]
Comment (fatpat) [reply cli/flags.go]: fixed

**Review by klauspost — APPROVED** [review]

**Pull request merged**